tests/garage/experiment/test_deterministic.py

```diff
@@ -38,6 +38,34 @@ def test_deterministic_tensorflow():
     assert np.allclose(rand_tensor, deterministic_tensor)
 
 
+def test_deterministic_tfp_seed_stream():
```

Comment:
this test is an okay unit tests but i think its helpfulness is very narrow for what's most important to test: whether we have achieved (a minimum level of) determinism.

can you also add an integration test which verifies that TF experiments are deterministic? rather than interacting with `get_tf_seed_stream()` directly, it would instead run an algorithm+environment (for very few steps) and ensure it's getting the same results.

Note that I don't think we've fully covered determinism in MuJoCo (or gym or Bullet or dm_control or Metaworld). Writing this test might tell us where our uncovered sources of randomness are. If you find them, we can make issues for them and address them later.